src/itemlistformaction.cpp

```diff
@@ -1075,6 +1085,7 @@ std::string ItemListFormAction::item2formatted_line(const ItemPtrPosPair& item,
 	fmt.register_fmt('n', item.first->unread() ? "N" : " ");
 	fmt.register_fmt('d', item.first->deleted() ? "D" : " ");
 	fmt.register_fmt('F', item.first->flags());
+	fmt.register_fmt('e', item.first->enclosure_url());
```

Comment:
This can be used for showing the enclosure URL for each item, but it can also be used to show some symbol if an article has an enclosure URL.

For example, using something like `%?e?<symbol>& ?` in the `articlelist-format` config:
```
articlelist-format "%4i %f %D  %?e?⇓& ? %6L %?T?|%-17T| ?%t"
```